code-review/README.md

```diff
@@ -84,6 +84,36 @@ experience, refactors the existing code). Then:
 * Sign off on the pull request by approving a [pull request
   review][pr-review-docs] in Github.
 
+### Migrations & Potential Downtime
+
+Due to our deployment process, we can encounter momentary `500` statuses when performing certain migration procedures. Namely, **removing** and **renaming** columns can result in these intermittent downtimes. Here's an example:
+
+1. We have an attribute `Model#unused_attribute`.
+2. A request comes in to remove this attribute and all references.
+3. We generate a migration to remove the attribute and references.
+4. We deploy the changes.
+5. _Uh oh..._
+
+During the migration period, just after the column is removed, the codebase changes have yet to rollover. Thus, if a user visits a page that references `Model#unused_attribute`, an exception will be raised, resulting in some potential downtimes.
```

Comment:
Hmm, I believe the order of operations is reversed, right?

1. Deploy
2. Migrate (via the `before_release` hook defined in .aptible.yml)

The issue happens between (1) and (2), if I'm understanding correctly

Reply:
@dleve123 "Deploy" (step 4) in this context means push to the Aptible remote, which migrates before rolling over the new code. I can update this to provide better clarity.

Reply:
Updating sounds good!